launcher/package-lock.json

```diff
@@ -0,0 +1,9963 @@
+{
```

Comment:
is `package-lock.json` file required?
I mean - VS Code uses `yarn.lock` files, Che-Code uses `yarn.lock` files for own extensions as well...

Reply:
I'm thinking how was this file appear in the project...
Thanks.